src/main/java/org/gridsuite/modification/utils/ModificationUtils.java

```diff
@@ -687,8 +689,36 @@ public ReportNode modifyBranchConnectivityAttributes(ConnectablePosition<?> conn
                                                          ReportNode connectivityReports) {
         List<ReportNode> reports = new ArrayList<>();
         processConnectivityPosition(connectablePosition, connectablePositionAdder, modificationInfos, branch.getNetwork(), reports);
-        modifyConnection(modificationInfos.getTerminal1Connected(), branch, branch.getTerminal1(), reports, ThreeSides.ONE);
-        modifyConnection(modificationInfos.getTerminal2Connected(), branch, branch.getTerminal2(), reports, ThreeSides.TWO);
+
+        List<NetworkModificationException> exceptions = new ArrayList<>();
+        // Pair for Side and Pair for update and terminal
+        List<Pair<ThreeSides, Pair<AttributeModification<Boolean>, Terminal>>> sidePairs =
+                List.of(
+                        Pair.create(ThreeSides.ONE, Pair.create(modificationInfos.getTerminal1Connected(), branch.getTerminal1())),
+                        Pair.create(ThreeSides.TWO, Pair.create(modificationInfos.getTerminal2Connected(), branch.getTerminal2()))
+                );
+        // We want informations for the both sides, not only the first in error
```

Comment:
information is a non count noun.
```suggestion
        // We want information for the both sides, not only the first in error
```